app/serializers/activity_serializer.rb

```diff
@@ -17,8 +17,8 @@
 #
 class ActivitySerializer
   include JSONAPI::Serializer
-  attributes :name, :content
 
+  attributes :name, :content
   attributes :image_url do |activity|
     activity.image.service_url
```

Comment:
Please add a verification to prevent errors:
```suggestion
    activity.image.service_url if activity.image.attached?
```